### DISCUSSION TITLE
holas buenas

### DISCUSSION THREAD

**Discussion by Wspmen** [coskundeniz/google-ad-clicker-premium#6] [General]
correctamente el nombre o, si incluyó una ruta de acceso, compruebe que dicha ruta es correcta e inténtelo de nuevo.
En línea: 1 Carácter: 1
+ pip install -r requirements.txt
+ ~~~
    + CategoryInfo          : ObjectNotFound: (pip:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

PS C:\Users\ATHLON 3000G\Desktop\google-ad-clicker-premium-main (2)\google-ad-clicker-premium-main>
 *  History restored 

PS C:\Users\ATHLON 3000G\Desktop\google-ad-clicker-premium-main (2)\google-ad-clicker-premium-main> 
 alguien sabe porque da este error???

**Comment by coskundeniz** [comment]
Please write in English.

**Reply by Wspmen** [reply]
Hi, does anyone know what the error is here or how to fix it?

**Comment by coskundeniz** [comment]
Can you put all the error output?

**Reply by Wspmen** [reply]
d-clicker-premium-main (2)\google-ad-clicker-p
remium-main\hooks.py", line 206
    }
    ^
SyntaxError: unmatched '}'
PS C:\Users\ATHLON 3000G\Desktop\google-ad-cli
cker-premium-main (2)\google-ad-clicker-premiu
m-main>
 *  History restored 

PS C:\Users\ATHLON 3000G\Desktop\google-ad-clicker-premium-main (2)\google-ad-clicker-premium-main>
this would be

**Reply by Wspmen** [reply]
PS C:\Users\ATHLON 3000G\Desktop\google-ad-clicker-premium-main (2)\google-ad-cl
icker-premium-main> & "C:/Users/ATHLON 3000G/AppData/Local/Programs/Python/Pytho
n313/python.exe" "c:/Users/ATHLON 3000G/Desktop/google-ad-clicker-premium-main (
2)/google-ad-clicker-premium-main/hooks.py"
Traceback (most recent call last):
  File "c:\Users\ATHLON 3000G\Desktop\google-ad-clicker-premium-main (2)\google-
ad-clicker-premium-main\hooks.py", line 4, in <module>
    import undetected_chromedriver
    import undetected_chromedriver
    import undetected_chromedriver
    import undetected_chromedriver
ModuleNotFoundError: No module named 'undetected_chromedriver'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):

Traceback (most recent call last):      
  File "c:\Users\ATHLON 3000G\Desktop\go
ogle-ad-clicker-premium-main (2)\google-
ad-clicker-premium-main\hooks.py", line 
    import undetected_chromedriv
er
ModuleNotFoundError: No module n
amed 'undetected_chromedriver'     import undetected_chromedri
ver
ModuleNotFoundError: No module 
named 'undetected_chromedriver    import undetected_chromedr
iver
ModuleNotFoundError: No module
 named 'undetected_chrome    import undetected_chr
omedriver
ModuleNotFoundError: No m
odule named 'undetected_c
    import undetected_chromedriver
ModuleNotFoundError: No module named 'undetec
    import undetected_chromedriver
ModuleNotFoundError: No module named 'undete
    import undetected_chromedriver
ModuleNotFoundError: No module named 'undet
    import undetected_chromedriver
ModuleNotFoundError: No module named 'undetect
ed_chromedriver'
PS C:\Users\ATHLON 3000G\Desktop\google-ad-cli
cker-premium-main (2)\google-ad-clicker-premiu
m-main>           > & "C:/Users/ATHLON 3000G/A
ppData/Local/Programs/Python/Python313/python.
exe" "c:/Users/ATHLON 3000G/Desktop/google-ad-
clicker-premium-main (2)/google-ad-clicker-pre
mium-main/hooks.py"
remium-main\hooks.py", line 202
    bash pip install undetected-chromedriver  
         ^^^
SyntaxError: invalid syntax
PS C:\Users\ATHLON 3000G\Desktop\google-ad-cli
cker-premium-main (2)\google-ad-clicker-premiu
m-main> & "C:/Users/ATHLON 3000G/AppData/Local
/Programs/Python/Python313/python.exe" "c:/Use
rs/ATHLON 3000G/Desktop/google-ad-clicker-prem
ium-main (2)/google-ad-clicker-premium-main/ho
oks.py"
remium-main\hooks.py", line 202
    bash pip install undetected-chromedriver  
         ^^^
SyntaxError: invalid syntax
PS C:\Users\ATHLON 3000G\Desktop\google-ad-cli
cker-premium-main (2)\google-ad-clicker-premiu
m-main> & "C:/Users/ATHLON 3000G/AppData/Local
/Programs/Python/Python313/python.exe" "c:/Use
rs/ATHLON 3000G/Desktop/google-ad-clicker-prem
ium-main (2)/google-ad-clicker-premium-main/ho
oks.py"
  File "c:\Users\ATHLON 3000G\Desktop\google-a
d-clicker-premium-main (2)\google-ad-clicker-p
remium-main\hooks.py", line 206
    }
    ^
SyntaxError: unmatched '}'
PS C:\Users\ATHLON 3000G\Desktop\google-ad-cli
cker-premium-main (2)\google-ad-clicker-premiu
m-main>
 *  History restored 

PS C:\Users\ATHLON 3000G\Desktop\google-ad-clicker-premium-main (2)\google-ad-clicker-premium-main>

**Reply by coskundeniz** [reply]
First of all, there isn't 206 lines in hooks.py file. If you added something, you may made a mistake.
Most importantly, you are trying to run it with python 3.13. Supported versions are 3.9 to 3.11 as written in explanations.
Make sure you have the right Python version and follow the steps in the How to setup section.

-----

If you can't manage to do it yourself, you can buy a support session.
https://coskundeniz.github.io/ad_clicker/

**Reply by Wspmen** [reply]
Hello, a question, what happens if I get an error when posting this?
1. python -m venv env
2. .\env\Scripts\activate
3. python -m pip install -r requirement

**Reply by coskundeniz** [reply]
You fix the error and continue.
